src/utils/tasks.js

```diff
@@ -0,0 +1,62 @@
+import TaskState from '@/model/TaskState.model'
+
+/**
+ * States used when comparing the parent with children nodes.
+ * @type {*[]}
+ */
+const orderedStates = [
+  TaskState.SUBMIT_FAILED,
+  TaskState.FAILED,
+  TaskState.EXPIRED,
+  TaskState.SUBMIT_RETRYING,
+  TaskState.RETRYING,
+  TaskState.RUNNING,
+  TaskState.SUBMITTED,
+  TaskState.READY,
+  TaskState.QUEUED,
+  TaskState.WAITING,
+  TaskState.HELD,
+  TaskState.SUCCEEDED,
+  TaskState.RUNAHEAD
+]
+
+/**
+ * States used when the parent is stopped.
```

Comment:
Perhaps `s/stopped/completed/` ?

Reply:
Fine by me, but I used stopped here because this states order was used when the `is_stopped` flag was set in the Python code: https://github.com/cylc/cylc-flow/blob/d66ae5c3ce8c749c8178d1cd53cb8c81d1560346/lib/cylc/task_state_prop.py#L101

Should I change it to completed?

Reply:
OK, not really important. If I understand the code correctly then I think "completed" is correct. At the moment the `isStopped` argument is always `false` anyway.